interfaces/story-dao-I.ts

```diff
@@ -2,13 +2,14 @@
  * @file Declares API for Stories related data access object methods
  */
 import Story from "../models/stories/story";
+import User from "../models/users/user";
```

Comment:
I dont think we are using `User` model anywhere in this class. We can remove this.

Reply:
removed